cdap-docs/build.sh

```diff
@@ -135,18 +131,18 @@ function build_docs_outer_level() {
   copy_source examples-manual     "Examples, Guides, and Tutorials"
```

Comment:
Should this be changed to a list, similar to line 153? The titles for these manuals are define in the common conf.py.